Library/Formula/knot.rb

```diff
@@ -39,6 +40,7 @@ def install
     inreplace "samples/Makefile", "install-data-local:", "disable-install-data-local:"
 
     system "make"
+    system "make", "check"
```

Comment:
Hold on gentlemen... the developers of Knot have added some tests to the configure script, such that it disables the "fast-parser" option if it's building on OSX with clang. They have a bug report open upstream for clang. Once it is fixed, they will reenable the fast parser. This means that when Knot 1.6.4 is out, we won't have to do anything special. The configure script itself will take care of things. And that method is portable - works for Fink and Macports too.